CONTRIBUTING.md

```diff
@@ -2,6 +2,17 @@
 
 # How to contribute to the Tribler project? #
 
+## Checking out the Stabilization Branch ##
+The stabilization branch `next` contains the most up to date bugfixes. If your issue cannot be reproduced there, it is most likely already fixed.
+
+To backup your Tribler installation and checkout the latest version of the stabilization branch, please perform the following steps.
+* Copy the `.Tribler` folder to a safe location on your system (for instance the desktop) Make sure to leave the original folder on its original location. This folder is located at `~/.Tribler/` (Linux/OSX) or `%APPDATA\.Tribler` (Windows).
+* Rename the `tribler` installation folder to `oldTribler`.
```

Comment:
I don't think that's a really good idea, let's just tell the user to keep the old installer handy in case they want to go back.